GroceryListService/Models/User.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GroceryListService.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; } // This might need to be changed for security reasons, but for now, we'll be okay
```

Comment:
You will need some form of password encryption, one option might be to use PasswordHasher